### PR TITLE
[UI/UX](Portfolio): Redesign Portfolio page

### PR DESCRIPTION
**Pull request by fantzulin**
## Types of changes

- **Other** (UI/UX)

## Description

Edit the style of the **First Impression** section and the **Rebalancer** section. Also, edit the style of the **Navbar** including `links` and the `Connect Wallet` button.

## Checklist:

- [ ]  "Financial Independence Retire Early"
- [ ] Must have slogan "Click Once, Retire Forever."
- [ ]  There is a "Invest Now!" call-to-action button in the center.
- [ ] Hope to place "APR" information conspicuously.
- [ ] Improve RWD effects.

### CODE REVIEW

File: pages/basePage.tsx [hunk 1]
```diff
@@ -1,79 +1,113 @@
...
+const { Header, Footer, Content } = Layout;
 interface BasePageProps {
   children: React.ReactNode;
 }
 
 const BasePage: NextPage<BasePageProps> = ({ children }) => {
   const { address } = useAccount();
+  const [menuShow, setMenuShow] = useState(false);
+  const handleMenuShow = () => {
+    setMenuShow(!menuShow);
```
Comment (david30907d) [inline pages/basePage.tsx:30]: Cool, thx for implementing this floating menu

File: pages/basePage.tsx [hunk 2]
```diff
@@ -1,79 +1,113 @@
...
+          <Header className={styles.header}>
+            <div className={styles.menuIcon}>
+              <MenuOutlined onClick={handleMenuShow}/>
+            </div>
+            <HeaderInner>
+              <div
+                className={
+                  menuShow
+                  ? styles.menuBarShow
+                  : styles.menuBar
```
Comment (david30907d) [inline pages/basePage.tsx:53]: Cool

File: pages/basePage.tsx [hunk 3]
```diff
@@ -1,79 +1,113 @@
...
+            href="https://discord.gg/sNsMmtsCCV"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            <img
+              src="../socialIcon/discord.png"
+              alt="discord"
+            />
+          </a>
+        </Footer>
```
Comment (david30907d) [inline pages/basePage.tsx:109]: Thx for refactoring the footer, the size of their pictures were different

File: pages/views/ExampleUI.jsx [hunk 4]
```diff
@@ -2,43 +2,149 @@ import React from "react";
...
+                    className={styles.btnInvest}
+                    style={
+                      isHover
+                      ? {backgroundColor: '#beed54', color: '#000000'}
+                      : {backgroundColor: 'transparent', color: '#beed54'}
+                    }
+                    onMouseEnter={handleMouseEnter}
+                    onMouseLeave={handleMouseLeave}
+                  >
+                    Invest Now!
```
Comment (david30907d) [inline pages/views/ExampleUI.jsx:113]: Thx for the beautiful landing page~

File: pages/views/ZapInButton.jsx [hunk 5]
```diff
@@ -355,42 +362,54 @@ const ZapInButton = () => {
...
+        <Space.Compact 
+          style={{
+            margin: '10px 0'
+          }}
+        >
+          {
+            selectBefore((value) => {
+              setChosenToken(value);
+            })
+          }
```
Comment (david30907d) [inline pages/views/ZapInButton.jsx:382]: Have you tested this one?

**Review by david30907d — APPROVED** [review]
LGTM

File: pages/views/ZapInButton.jsx (hunk 5 above)
Comment (fantzulin) [reply pages/views/ZapInButton.jsx:382]: I haven't tested this.

**Pull request merged**